CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add strategy for American Government answer key (minor)
 * Add `BakeReferences` direction (minor)
 * Fix xmlns string replacement done in PR #209 (minor)
+* Move title above metadata in `BakeSuggestedReading` direction (minor)
```

Comment:
This could arguably be "major" -- in the visible materials nothing moves, but other recipe code could break b/c of the reordering of elements.